recipes/openssl/1.x.x/conanfile.py

```diff
@@ -241,17 +232,53 @@ def validate(self):
 
     def build_requirements(self):
         if self._settings_build.os == "Windows":
-            if not self._win_bash:
-                self.build_requires("strawberryperl/5.30.0.1")
-            if not self.options.no_asm and not tools.which("nasm"):
-                self.build_requires("nasm/2.15.05")
-        if self._win_bash and not tools.get_env("CONAN_BASH_PATH"):
+            if not self.win_bash:
```

Comment:
This is the new way for future readers https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html?highlight=win_bash#windows-subsystems